source/shared/cpp/ObjectModel/BaseCardElement.h

```diff
@@ -49,7 +49,6 @@ class BaseCardElement
     static Json::Value SerializeSelectAction(const std::shared_ptr<BaseActionElement> selectAction);
 
 private:
-    static const std::unordered_map<ActionType, std::function<std::shared_ptr<BaseActionElement>(const Json::Value&)>, EnumHash> ActionParsers;
     CardElementType m_type;
```

Comment:
Who was using ActionParsers?

Reply:
It's a remnant from how we defined parsers before we had custom parsers, and is no longer in use.